en/docs/learn/analytics/analytics-event-streams-and-aggregations.md

```diff
@@ -123,7 +123,7 @@ This stream contains the API invocation with throttle information. Throttling ca
 
 The summarized tables are stored in the API-M Analytics internal storage. All events are stored in tables based on granularity (`SECONDS`, `MINUTES`, `HOURS`, `DAYS`, `MONTHS`, and `YEARS`). Each such table contains the aggregation of stream data per granularity level. For example, each record of the `_HOURS` table contains the aggregate of the events that arrived each hour.
 
-It is important to note that directly querying the tables is not recommended and instead you can use the [WSO2 Streaming Integrator REST APIs](https://ei.docs.wso2.com/en/latest/streaming-integrator/setup/setting-up-physical-databases/#managing-stored-data-via-rest-apis) for retrieving data from aggregations.
+It is important to note that directly querying the tables is not recommended and instead you can use the WSO2 Streaming Integrator REST APIs for retrieving data from aggregations. For more information, see [Integrating Data Stores in Streaming Integration](https://ei.docs.wso2.com/en/7.1.0/streaming-integrator/examples/integrating-stores/).
```

Comment:
the link should be https://ei.docs.wso2.com/en/latest/streaming-integrator/examples/integrating-stores/ isn't it

Reply:
It is best to point to a specific version of the doc always.